tests/core/server/test_rate_limits.py

```diff
@@ -23,11 +23,11 @@ async def test_too_many_messages(self):
         # Too many messages
         r = RateLimiter(incoming=True)
         new_tx_message = make_msg(ProtocolMessageTypes.new_transaction, bytes([1] * 40))
-        for i in range(3000):
+        for i in range(4900):
```

Comment:
Is this related to the rest of the changes in some way?

Reply:
No it is not, it was just a flaky test that needed a fix.